docs/getting-started/db.md

```diff
@@ -4,15 +4,18 @@ title: Preparing the Database
 
 # Preparing the database
 
-We need to create our database, load the database schema, and load seed
-data. This can be accomplished with:
+The next step is to create and prepare the database. Thankfully, dev.to is a
```

Comment:
I wonder if we should use "DEV" here instead of "dev.to"

Reply:
I agree!